### PR TITLE
began working on organizationCreate page

### PR DESCRIPTION
**Pull request by PatrickMetsch**
Created the bare bones of the createOrganization page.

Everything is fine until it's time to add the new organization object to the organizations array (index.tsx, line 96). I've console.logged to ensure the information is flowing properly, and it is. I've proven that every time I click "submit" (createOrganization.tsx, line 21), that the object is added to the array. However, I cannot get the new array to reflect the newly added organization when I navigate back to the organizationSearch page and try to locate the new organization.

I've tried `.push`, `'.concat`, and decided to stick with the ES6-ier syntax that I've submitted. 

Lemme know what you think.

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -69,6 +69,34 @@ const renderApp = (state) => {
         });
     }
 
+    const updateNewOrgTitle = (newOrgTitle) => {
+        renderApp({
+            ...state,
+            newOrgTitle
```
Comment (tommysullivan) [inline src/index.tsx]: Here u add a property called newOrgTitle to the state; but it needs to be inside the organizations array

File: src/index.tsx [hunk 2]
```diff
@@ -117,7 +172,10 @@ const initialState = {
     inDevMode,
     searchBarValue: '',
     searchBarSuggestions: [],
-    shouldShowSearch: true
+    shouldShowSearch: true,
+    newOrgTitle: '',
```
Comment (tommysullivan) [inline src/index.tsx]: Not yet getting why these are here maybe cuz u want to track what’s being entered prior to the new organization being made via form submission. When does the new org go into the organizations list tho?

**Comment by tommysullivan** [comment]
@PatrickMetsch any updates here?

**Comment by PatrickMetsch** [comment]
I have allocated some time for tomorrow evening.

**Comment by tommysullivan** [comment]
cool fyi u will need a decent rebase cuz we merged 3 prs and one more on the way shortly

**Comment by tommysullivan** [comment]
fyi @PatrickMetsch i created another PR off of the "rebase" version of this branch so as to not trample on you. Let me know, one of these PRs can be closed. But i don't wanna lose any of your changes you may have made. 

https://github.com/heartpoints/heartpoints.org/pull/134

happy to help get the changes from either branch to the other if need be just lmk :)

excited for "view org"

**Comment by tommysullivan** [comment]
Closing this!

**Pull request closed without merging**